src/benchmarklib/abstract_query_generator.hpp

```diff
@@ -23,6 +23,10 @@ class AbstractQueryGenerator {
   // Generates a SQL string for a given (zero-indexed) query id
   virtual std::string build_query(const QueryID query_id) = 0;
 
+  // Some benchmarks might select random parameters (e.g., TPC-H). For test purposes, this method returns a constant
```

Comment:
I don't know what "test purposes" are. "For verification runs of a benchmark, ..." would be clearer to me.
(minor: why not const?)

Reply:
Added comment.
Regarding the const: The default implementation uses `build_query`, which cannot be const.